packages/fxa-settings/src/pages/AccountRecoveryConfirmKey/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import AccountRecoveryConfirmKey, { AccountRecoveryConfirmKeyProps } from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { MOCK_SERVICE_NAME } from './mocks';
+
+export default {
+  title: 'pages/AccountRecoveryConfirmKey',
+  component: AccountRecoveryConfirmKey,
+} as Meta;
```

Comment:
It really makes me laugh that we call this `Meta` for a facebook product

Reply:
That's so meta!